couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -43,6 +45,7 @@ def __init__(self, name, init_config, instances):
         super(Couchbase, self).__init__(name, init_config, instances)
 
         self._sync_gateway_url = self.instance.get('sync_gateway_url', None)
+        self._index_stats_url = self.instance.get('index_stats_url', None)
```

Comment:
```suggestion
        self._index_stats_url = self.instance.get('index_stats_url')
```

get() defaults to None already